src/victory-legend/victory-legend.js

```diff
@@ -229,22 +244,22 @@ export default class VictoryLegend extends React.Component {
   }
 
   renderGroup(props, children) {
-    const { groupComponent, height, width, standalone, x, y } = props;
-    const groupProps = { role: "presentation" };
+    const { groupComponent, height, parentStyles, standalone, width, x, y } = props;
+    let groupProps = { role: "presentation" };
 
     if (!standalone) {
-      Object.assign(groupProps, { height, width, x, y });
+      groupProps = merge(groupProps, { height, width, x, y, style: parentStyles });
```

Comment:
good catch! i switched this when i was merging nested style objects but then later refactored and forgot to switch back